app/test/controllers/Todo/index.js

```diff
@@ -0,0 +1,73 @@
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const Register = require('../../helpers/register');
+const TodoFactory = require('../../factories/todo');
+const truncateDatabase = require('../../helpers/truncate');
+
+const app = require('../../../index');
+const request = require('supertest')(app);
+
+let todos = [];
+let todosAnotherUser = [];
+let loggerUserId;
+let loggedUserToken;
+
+describe('Todos', () => {
+    before(async () => {
+        await truncateDatabase();
+
+        const { user, token } = await Register(request);
+        loggedUserToken = token;
+        loggerUserId = user.id;
+
+        const { user: anotherUser } = await Register(request);
+
+        todos.push(await TodoFactory.create({ userId: loggerUserId }));
+        todos.push(await TodoFactory.create({ userId: loggerUserId }));
+        todos.push(await TodoFactory.create({ userId: loggerUserId }));
+        todosAnotherUser.push(
+            await TodoFactory.create({ userId: anotherUser.id })
+        );
+    });
+
+    describe('GET /todos', () => {
+        it('Returns OK and fetches all todos that belongs to user as Logged in', async () => {
```

Comment:
Please do everything as in ERP system. 
`returns OK with all todos as USER`

Reply:
this demo is very very old, the tests also, we changed all standards and we should keep everything as in ERP system